docs/Release_Management.md

```diff
@@ -44,6 +44,18 @@ This project strictly follows [semantic versioning](https://semver.org/spec/v2.0
 
 - Any `fixes` or `patches` should be merged to main and then `cherry pick` to the release branch.
 
+## Supported Releases
+
+Applicable fixes, including security fixes, may be cherry-picked into the release branch, depending on severity and feasibility. Patch releases are cut from that branch as needed.
+
+We expect users to stay reasonably up-to-date with the versions of Secrets Store CSI Driver they use in production, but understand that it may take time to upgrade. We expect users to be running approximately the latest patch release of a given minor release and encourage users to upgrade as soon as possible.
+
+We expect to "support" n (current) and n-1 major.minor releases. "Support" means we expect users to be running that version in production. For example, when v1.3.0 comes out, v1.1.x will no longer be supported for patches and we encourage users to upgrade to a supported version as soon as possible.
+
+## Supported Kubernetes Versions
+
+Secrets Store CSI Driver is assumed to be compatible with n-4 versions of the latest stable Kubernetes release per [Kubernetes Supported Versions policy](https://kubernetes.io/releases/version-skew-policy/). If you choose to use Secrets Store CSI Driver with a version of Kubernetes that it does not support, you are using it at your own risk.
```

Comment:
I'm not seeing how the version skep policy link translates to n-4? is that the correct link?

Reply:
> The Kubernetes project maintains release branches for the most recent three minor releases (1.22, 1.21, 1.20). Kubernetes 1.19 and newer receive approximately 1 year of patch support. Kubernetes 1.18 and older received approximately 9 months of patch support.

The current release is 1.22 and 1.19 above is still supported.

Reply:
Ah, I took that as that the intent was to support 3 minor releases and that 1.19 was a special case where it transitioned to a 12 month support flow, but that going forward it will only be 3 branches.

It might be better to phrase it as `Secrets Store CSI Driver will maintain support for all actively supported Kubernetes minor releases`, so that it stays in sync if K8s support periods change